src/metabase/upload/impl.clj

```diff
@@ -539,7 +539,7 @@
         table             (sync/create-table! db {:name         table-name
                                                   :schema       (not-empty schema)
                                                   :display_name display-name})
-        _set_is_upload    (t2/update! :model/Table (:id table) {:is_upload true})
+        _set_is_upload    (t2/update! :model/Table (:id table) {:is_upload true, :data_authority :authoritative})
```

Comment:
Not really my area, but based on the definitions of the enum values in `warehouse_schema.models.table/writable-data-authority-types` I would have expected this to be `:ingested`, ie. the data came from elsewhere and might get replaced.

If `:ingested` means something more like _this comes from another database as a nightly dump_, then I guess this tracks. We don't expect the CSV upload to ever change.